src/sentry/tasks/post_process.py

```diff
@@ -1367,7 +1378,10 @@ def detect_new_escalation(job: PostProcessJob):
     ):
         return
     group_age_hours = (datetime.now() - group.first_seen).total_seconds() / 3600
-    if group_age_hours >= MAX_NEW_ESCALATION_AGE_HOURS or group.substatus != GroupSubStatus.NEW:
+    has_valid_status = group.substatus == GroupSubStatus.NEW or (
+        group.status == GroupStatus.IGNORED and group.substatus == GroupSubStatus.UNTIL_ESCALATING
```

Comment:
rn there aren't cases where we'd have `GroupSubStatus.UNTIL_ESCALATING` without `GroupStatus.IGNORED` but i thought it was worth checking for status any way in case we decided to use the substatus for other things